scripts/install_python.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+sudo apt install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev
+
+curl https://pyenv.run | bash
+
+echo 'export PATH="${HOME}/.pyenv/bin:${PATH}"' >> ~/.bashrc
+echo 'eval "$(pyenv init -)"' >> ~/.bashrc
+echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.bashrc
+export PATH="${HOME}/.pyenv/bin:${PATH}"
+source ~/.bashrc
```

Comment:
I think we'll probably want get clear consent before editing the user's `.bashrc`.  We could either have a script called (e.g.) `load_pyenv_on_startup.sh`, or just expect the user to set up python themselves.  I'm inclined towards the latter -- taking on responsibility for Python setup seems like too much responsibility for this repo.

Reply:
Good point. I can remove this.

Reply:
I have added a clear dialog asking for consent. If it is denied, the instructions will only be printed to stdout, but no action will be taken. I also create a backup of `.bashrc` before applying any changes even if we receive consent. I hope this addresses your concerns.

Reply:
yeah this is great